.github/workflows/Compile.yml

```
@@ -5,21 +5,46 @@ on:
+          cache: npm
+          cache-dependency-path: facade-app/package-lock.json
+
+      - name: Build and test React app
+        working-directory: facade-app
+        run: |
+          npm ci
+          npm run build
+          npm run test -- --coverage
+
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.12)

[error] the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)